app/cdap/components/ParsingDrawer/Components/ParsingPopupBody/index.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, InputLabel } from '@material-ui/core';
+import React, { ChangeEvent, useEffect } from 'react';
+import { useStyles } from '../../styles';
+import InputCheckbox from '../InputCheckbox';
+import InputSelect from '../InputSelect';
+import { CHAR_ENCODING_OPTIONS, FORMAT_OPTIONS } from './parsingOptions';
+import T from 'i18n-react';
+
+export default function({ values, changeEventListener }) {
+  const classes = useStyles();
+
+  const { format, fileEncoding, enableQuotedValues, skipHeader } = values;
+  let selectedFormatValue = [];
+  let selectedEncodingValue = [];
+
+  useEffect(() => {
+    selectedFormatValue = FORMAT_OPTIONS.filter((i) => i.value === format);
+  }, [format]);
+
+  useEffect(() => {
+    selectedEncodingValue = CHAR_ENCODING_OPTIONS.filter((i) => i.value === fileEncoding);
+  }, [fileEncoding]);
+
+  return (
+    <Box>
+      <Box className={[classes.formFieldWrapperStyles, classes.marginBottomStyles].join(' ')}>
+        <InputLabel id="label" className={classes.labelTextStyles}>
+          {T.translate('features.WranglerNewParsingDrawer.format')}
+        </InputLabel>
+        <InputSelect
+          classes={{ icon: classes.selectIconStyles, select: classes.selectStyles }}
+          className={classes.selectFieldStyles}
+          optionClassName={{ root: classes.optionStyles }}
+          fullWidth
+          defaultValue={FORMAT_OPTIONS[0].value}
+          value={selectedFormatValue[0]?.value}
+          onChange={(event: ChangeEvent<HTMLSelectElement>) =>
+            changeEventListener(event.target.value, 'format')
+          }
+          options={FORMAT_OPTIONS}
+        />
+      </Box>
+
+      <Box className={[classes.formFieldWrapperStyles, classes.marginBottomStyles].join(' ')}>
```

Comment:
we can use string interpolation here
{${class1} ${class2}}

Reply:
Done